tests/test_ble_state_manager.py

```
@@ -146,7 +179,9 @@ def worker(worker_id):
                         success = manager.transition_to(ConnectionState.DISCONNECTED)
                     results.append((worker_id, i, success, manager.state.value))
                     time.sleep(0.001)  # Small delay to increase contention
-            except Exception as e:
+            except (
+                Exception
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)